src/main/java/com/google/alpollo/ConfigHelper.java

```diff
@@ -0,0 +1,45 @@
+package com.google.alpollo;
+
+import com.google.gson.Gson;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.Objects;
+
+/**
+ * Helps to load configuration file and retrieve data from it.
+ */
+final public class ConfigHelper {
+  private static final String CONFIG_FILE_NAME = "config.json";
+  private static final Gson gson = new Gson();
+
+  /**
+   * Class which represents the content of configuration file,
+   * used only to get Json object from file.
+   */
+  private static final class ConfigInfo {
+    private String projectID;
+
+    public String getProjectID() {
+      return projectID;
+    }
+  }
+
+  private ConfigHelper() {
+    throw new RuntimeException("Instantiation of ConfigHelper is not allowed!");
+  }
+
+  /**
+   * Retrieves projectID from configuration file.
+   * @return projectID; {@code null} if configuration file wasn't found
+   * or it was incorrect/didn't have projectID field
+   */
+  public static String getProjectID() {
+    try (InputStream inputStream = ClassLoader.getSystemResourceAsStream(CONFIG_FILE_NAME)) {
+      final Reader reader = new InputStreamReader(Objects.requireNonNull(inputStream));
```

Comment:
I should have been more specific, the reader also needs to be declared inside the try (), as it is also a Closeable

Reply:
sure, I forgot(